apps/users-service/src/handlers/fetch-me.ts

```diff
@@ -0,0 +1,57 @@
+import Boom from '@hapi/boom';
+import got from 'got';
+import Intercept from 'apr-intercept';
+import { config as authConfig } from '@asap-hub/auth';
+import { framework as lambda } from '@asap-hub/services-common';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+import Users from '../controllers/users';
+import * as auth0 from '../entities/auth0';
+
+const validateUser = async (
+  request: lambda.Request,
+): Promise<auth0.UserInfo> => {
+  const headers = request.headers as {
+    authorization: string;
+  };
+
+  if (!headers.authorization) {
+    throw Boom.unauthorized();
+  }
+
+  const [type, token] = headers.authorization.split(' ');
+
+  if (type.toLowerCase() !== 'bearer') {
+    throw Boom.forbidden();
+  }
+
+  const [err, res] = await Intercept(
+    got(`https://${authConfig.domain}/userinfo`, {
+      headers: {
+        Authorization: `Bearer ${token}`,
+        'Content-Type': 'application/json',
+      },
+    }).json<auth0.UserInfo>(),
+  );
+
+  if (err) {
+    throw Boom.forbidden('Forbidden', {
```

Comment:
Can (and do we want to) forward the status code from auth0 userinfo? I think probably yes - if they give a 4xx it's probably something about the token the client provided, we want to pass that through, if it's 5xx it's probably some Auth0 outage or whatever and we don't want the client to think their authentication is just invalid.

Reply:
That's right, let me add a test to cover that scenario.

Reply:
The fix is done. But I'll need someones help on fixing the typescript error.
![image](https://user-images.githubusercontent.com/6756384/87535169-f7024c00-c68e-11ea-9ab0-e2181d88d588.png)

Reply:
I can TAL (not sure when exactly because of upcoming flight)

Reply:
I think I managed to work around it 😉 For now it's ok!